diff -urN ucsim-0.2.38.org/Makefile ucsim-0.2.38/Makefile
--- ucsim-0.2.38.org/Makefile	Fri Feb 16 17:16:08 2001
+++ ucsim-0.2.38/Makefile	Thu Mar 29 18:33:25 2001
@@ -33,7 +32,7 @@
 install:
 	$(MAKE) -f main.mk install
 	@for pkg in $(PKGS); do\
-	  $(MAKE) -C $$pkg install ;\
+	  $(MAKE) -C $$pkg DESTDIR=$(DESTDIR) install ;\
 	done
 
 
diff -urN ucsim-0.2.38.org/avr.src/Makefile.in ucsim-0.2.38/avr.src/Makefile.in
--- ucsim-0.2.38.org/avr.src/Makefile.in	Fri Feb 16 17:16:08 2001
+++ ucsim-0.2.38/avr.src/Makefile.in	Thu Mar 29 18:20:44 2001
@@ -56,13 +56,13 @@
 # Compiling and installing everything and runing test
 # ---------------------------------------------------
 install: all installdirs
-	$(INSTALL) -s savr $(bindir)
+	$(INSTALL) savr $(DESTDIR)$(bindir)
 
 
 # Deleting all the installed files
 # --------------------------------
 uninstall:
-	rm -f $(bindir)/savr
+	rm -f $(DESTDIR)$(bindir)/savr
 
 
 # Performing self-test
@@ -78,7 +78,7 @@
 # Creating installation directories
 # ---------------------------------
 installdirs:
-	test -d $(bindir) || $(INSTALL) -d $(bindir)
+	test -d $(DESTDIR)$(bindir) || $(INSTALL) -d $(DESTDIR)$(bindir)
 
 
 # Creating dependencies
diff -urN ucsim-0.2.38.org/configure.in ucsim-0.2.38/configure.in
--- ucsim-0.2.38.org/configure.in	Fri Feb 16 17:16:08 2001
+++ ucsim-0.2.38/configure.in	Thu Mar 29 18:28:53 2001
@@ -38,8 +38,8 @@
 # Checking for header files.
 # ===========================================================================
 
-if test -d /stuff/include; then
-  CPPFLAGS="$CPPFLAGS -I/stuff/include"
+if test -d /usr/include/ncurses; then
+  CPPFLAGS="$CPPFLAGS -I/usr/include/ncurses"
 fi
 
 AC_HEADER_STDC
@@ -105,16 +105,16 @@
 AC_CHECK_LIB(panel,panel_above,
 panel_ok="yes"
 curses_ok="yes"
-CURSES_LIBS="-lpanel -lcurses"
+CURSES_LIBS="-lpanel -lncurses"
 ,
 panel_ok="no"
-AC_CHECK_LIB(curses,nl,
+AC_CHECK_LIB(ncurses,nl,
 curses_ok="yes"
-CURSES_LIBS="-lcurses"
+CURSES_LIBS="-lncurses"
 ,
 ncurses_ok="no")
 ,
--lcurses)
+-lncurses)
 AC_SUBST(panel_ok)
 AC_SUBST(curses_ok)
 AC_SUBST(CURSES_LIBS)
diff -urN ucsim-0.2.38.org/doc/Makefile.in ucsim-0.2.38/doc/Makefile.in
--- ucsim-0.2.38.org/doc/Makefile.in	Fri Feb 16 17:16:08 2001
+++ ucsim-0.2.38/doc/Makefile.in	Thu Mar 29 18:20:44 2001
@@ -34,16 +34,16 @@
 # Compiling and installing everything and runing test
 # ---------------------------------------------------
 install: all installdirs
-	cp *.html $(docdir)
-	cp *.gif $(docdir)
-	cp *.jpg $(docdir)
-	cp *.fig $(docdir)
+	cp *.html $(DESTDIR)$(docdir)
+	cp *.gif $(DESTDIR)$(docdir)
+	cp *.jpg $(DESTDIR)$(docdir)
+	cp *.fig $(DESTDIR)$(docdir)
 
 
 # Deleting all the installed files
 # --------------------------------
 uninstall:
-	rm -f $(docdir)/*
+	rm -f $(DESTDIR)$(docdir)/*
 
 
 # Performing self-test
@@ -59,7 +59,7 @@
 # Creating installation directories
 # ---------------------------------
 installdirs:
-	test -d $(docdir) || $(INSTALL) -d $(docdir)
+	test -d $(DESTDIR)$(docdir) || $(INSTALL) -d $(DESTDIR)$(docdir)
 
 
 # Creating dependencies
diff -urN ucsim-0.2.38.org/gui.src/Makefile.in ucsim-0.2.38/gui.src/Makefile.in
--- ucsim-0.2.38.org/gui.src/Makefile.in	Fri Feb 16 17:16:08 2001
+++ ucsim-0.2.38/gui.src/Makefile.in	Thu Mar 29 18:20:44 2001
@@ -66,7 +66,7 @@
 # ---------------------------------------------------
 install:
 	@for pkg in $(PKGS); do\
-	  cd $$pkg && $(MAKE) install ; cd ..;\
+	  cd $$pkg && $(MAKE) DESTDIR=$(DESTDIR) install ; cd ..;\
 	done
 
 
@@ -74,7 +74,7 @@
 # --------------------------------
 uninstall:
 	@for pkg in $(PKGS); do\
-	  cd $$pkg && $(MAKE) uninstall ; cd ..;\
+	  cd $$pkg && $(MAKE) DESTDIR=$(DESTDIR) uninstall ; cd ..;\
 	done
 
 
@@ -98,7 +98,7 @@
 # ---------------------------------
 installdirs:
 	@for pkg in $(PKGS); do\
-	  cd $$pkg && $(MAKE) installdirs ; cd ..;\
+	  cd $$pkg && $(MAKE) DESTDIR=$(DESTDIR) installdirs ; cd ..;\
 	done
 
 
diff -urN ucsim-0.2.38.org/gui.src/serio.src/Makefile.in ucsim-0.2.38/gui.src/serio.src/Makefile.in
--- ucsim-0.2.38.org/gui.src/serio.src/Makefile.in	Fri Feb 16 17:16:08 2001
+++ ucsim-0.2.38/gui.src/serio.src/Makefile.in	Thu Mar 29 18:20:44 2001
@@ -48,13 +48,13 @@
 # Compiling and installing everything and runing test
 # ---------------------------------------------------
 install: all installdirs
-	$(INSTALL) -s serialview $(bindir)
+	$(INSTALL) serialview $(DESTDIR)$(bindir)
 
 
 # Deleting all the installed files
 # --------------------------------
 uninstall:
-	rm -f $(bindir)/serialview
+	rm -f $(DESTDIR)$(bindir)/serialview
 
 
 # Performing self-test
@@ -70,7 +70,7 @@
 # Creating installation directories
 # ---------------------------------
 installdirs:
-	test -d $(bindir) || $(INSTALL) -d $(bindir)
+	test -d $(DESTDIR)$(bindir) || $(INSTALL) -d $(DESTDIR)$(bindir)
 
 
 # Creating dependencies
diff -urN ucsim-0.2.38.org/main_in.mk ucsim-0.2.38/main_in.mk
--- ucsim-0.2.38.org/main_in.mk	Fri Feb 16 17:16:08 2001
+++ ucsim-0.2.38/main_in.mk	Thu Mar 29 18:20:44 2001
@@ -51,10 +51,10 @@
 # Deleting all the installed files
 # --------------------------------
 uninstall:
-	rm -f $(bindir)/s51
-	rm -f $(bindir)/savr
-	rm -f $(bindir)/serialview
-	rm -f $(bindir)/portmon
+	rm -f $(DESTDIR)$(bindir)/s51
+	rm -f $(DESTDIR)$(bindir)/savr
+	rm -f $(DESTDIR)$(bindir)/serialview
+	rm -f $(DESTDIR)$(bindir)/portmon
 
 
 # Performing self-test
diff -urN ucsim-0.2.38.org/s51.src/Makefile.in ucsim-0.2.38/s51.src/Makefile.in
--- ucsim-0.2.38.org/s51.src/Makefile.in	Fri Feb 16 17:16:08 2001
+++ ucsim-0.2.38/s51.src/Makefile.in	Thu Mar 29 18:20:44 2001
@@ -59,14 +59,14 @@
 # Compiling and installing everything and runing test
 # ---------------------------------------------------
 install: all installdirs
-	if test -f s51.exe; then $(INSTALL) -s s51.exe $(bindir);fi
-	if test -f s51; then $(INSTALL) -s s51 $(bindir);fi
+	if test -f s51.exe; then $(INSTALL) s51.exe $(DESTDIR)$(bindir);fi
+	if test -f s51; then $(INSTALL) s51 $(DESTDIR)$(bindir);fi
 
 
 # Deleting all the installed files
 # --------------------------------
 uninstall:
-	rm -f $(bindir)/s51
+	rm -f $(DESTDIR)$(bindir)/s51
 
 
 # Performing self-test
@@ -82,7 +82,7 @@
 # Creating installation directories
 # ---------------------------------
 installdirs:
-	test -d $(bindir) || $(INSTALL) -d $(bindir)
+	test -d $(DESTDIR)$(bindir) || $(INSTALL) -d $(DESTDIR)$(bindir)
 
 
 # Creating dependencies
diff -urN ucsim-0.2.38.org/z80.src/Makefile.in ucsim-0.2.38/z80.src/Makefile.in
--- ucsim-0.2.38.org/z80.src/Makefile.in	Fri Feb 16 17:16:08 2001
+++ ucsim-0.2.38/z80.src/Makefile.in	Thu Mar 29 18:20:44 2001
@@ -55,13 +55,13 @@
 # Compiling and installing everything and runing test
 # ---------------------------------------------------
 install: all installdirs
-	$(INSTALL) -s sz80 $(bindir)
+	$(INSTALL) sz80 $(DESTDIR)$(bindir)
 
 
 # Deleting all the installed files
 # --------------------------------
 uninstall:
-	rm -f $(bindir)/sz80
+	rm -f $(DESTDIR)$(bindir)/sz80
 
 
 # Performing self-test
@@ -77,7 +77,7 @@
 # Creating installation directories
 # ---------------------------------
 installdirs:
-	test -d $(bindir) || $(INSTALL) -d $(bindir)
+	test -d $(DESTDIR)$(bindir) || $(INSTALL) -d $(DESTDIR)$(bindir)
 
 
 # Creating dependencies
